### PR TITLE
Updating sample.servlet with Gradle

### PR DESCRIPTION
**Pull request by dshimo**
Review to merge!

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
-sudo: false
 language: java
-jdk: oraclejdk7
-script: mvn test
+jdk:
+    - oraclejdk7
```
Comment (cthigh) [inline .travis.yml]: should be oraclejdk8

File: README.md [hunk 2]
```diff
@@ -3,33 +3,69 @@ Ferret [![Build Status](https://travis-ci.org/WASdev/sample.ferret.svg?branch=ma
...
-6. Go to: [http://localhost:9080/ferret](http://localhost:9080/ferret)
+5. Right click on the project and go to *Properties > Project Facets* and select *Dynamic Web Module* (if not already selected).
+6. Deploy the sample into Liberty server. Right click on the *servlet* sample and select *Run As -> Run on Server* option. Find and select the Liberty profile server and press *Finish*. 
+7. Go to: [http://localhost:9080/ferret](http://localhost:9080/ferret)
 
 ## Running with Maven
 
-This project can be build with [Apache Maven](http://maven.apache.org/). The project uses [Liberty Maven Plug-in][] to automatically download and install Liberty profile runtime from the [Liberty repository](https://developer.ibm.com/wasdev/downloads/). Liberty Maven Plug-in is also used to create, configure, and run the application on the Liberty server. 
+This project can be built with Apache Maven. The project uses Liberty Maven Plug-in to automatically download and install Liberty with Java EE Web Profile 7 runtime from Maven Central. Liberty Maven Plug-in is also used to create, configure, and run the application on the Liberty server.
```
Comment (cthigh) [inline README.md]: I like having the links that you removed.

File: README.md [hunk 3]
```diff
@@ -3,33 +3,69 @@ Ferret [![Build Status](https://travis-ci.org/WASdev/sample.ferret.svg?branch=ma
...
     $ mvn liberty:run-server
     ```
 
 Once the server is running, the application will be available under [http://localhost:9080/ferret](http://localhost:9080/ferret).
 
+Please refer to the [ci.maven](http://github.com/WASDev/ci.maven) repository for documentation about using the Liberty Maven Plug-in.
+
+## Running with Gradle
+
+This project can also be built and run with Gradle. The provided `build.gradle` file applies the Liberty Gradle Plug-in and is configured to automatically download and install Liberty with Java EE Web Profile 7 runtime from Maven Central. The Liberty Gradle Plug-in has built-in tasks that can be used to create, configure, and run the application on the Liberty server.
```
Comment (cthigh) [inline README.md]: if you add the links back for Maven, add the equivalent gradle ones.

File: build.gradle [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        maven {
+            name = 'Sonatype Nexus Snapshots'
+            url = 'https://oss.sonatype.org/content/repositories/snapshots/'
+        }
+    }
+    dependencies {
+        classpath 'net.wasdev.wlp.gradle.plugins:liberty-gradle-plugin:1.1-SNAPSHOT'
```
Comment (cthigh) [inline build.gradle]: move up to the 2.0-SNAPSHOT

File: README.md (hunk 2 above)
Comment (dshimo) [reply README.md]: accident i think

**Comment by cthigh** [comment]
need pom.xml version update for the plugin.

**Pull request merged**